bin/simple-local-server.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+brew services start elasticsearch
+brew services start redis
+brew services start mysql
+
+yq -yi '.development.host="127.0.0.1" | .test.host="127.0.0.1"' config/database.yml
```

Comment:
You should be able to use env variables too, no?
```
development:
  host: <%= ENV['DATABASE_HOST'] %>
```

Reply:
Did not think of that, will try.